### DISCUSSION TITLE
Breadcrumbs should include resource group

### DISCUSSION THREAD

**Discussion by dmason30** [laravel/nova-issues#5082] [Ideas & Feature Requests]
- Nova Version: 4.19.2

### Description:
I think the below should display as Resources > Admin > Customers.

![image](https://user-images.githubusercontent.com/20278756/202455408-0ff6c55a-507a-4a88-a439-1395069ed9a7.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:
```php
class CustomerResource extends Resource
{
    public static function group()
    {
        return __('Admin');
    }
```

**Comment by sirikkoster** [comment]
Any progress ?

**Comment by ao-jhelmich** [comment]
Bumping it! This would be great
